plugin-api.d.ts

```diff
@@ -1143,6 +1145,15 @@ interface LinkUnfurlNode extends OpaqueNodeMixin, SceneNodeMixin {
   clone(): LinkUnfurlNode
 }
 
+interface MediaData {
+  hash: string;
+}
+interface MediaNode extends OpaqueNodeMixin, SceneNodeMixin {
```

Comment:
No need to add SceneNodeMixin here since OpaqueNodeMixin extends that.

We should also make sure that this interface contains the resize and resizeWithoutConstraints functions, right?

Reply:
Oops I realized that `resizeWithoutConstraints` will do the same thing as `resize` so will remove it here and in the other pr 🙏